### ISSUE TITLE
Adding False Friends Dataset to German MTEB

### ISSUE THREAD

**Issue by achibb** [embeddings-benchmark/mteb#331]
Hi all,

dataset:
https://huggingface.co/datasets/aari1995/false_friends_en_de

[associated paper](https://drive.google.com/file/d/1jgq0nBnV-UiYNxbKNrrr2gxDEHm-DMKH/view?usp=share_link)

this dataset can be used to test german and especially multilingual models for their understanding capabilities of "false-friends".
false friends are words that sound the same / spelled similarly but mean different in different languages.

Example false friend:
german: boot (boat)
english: boot (shoe)

The evaluation is the following:

input sentence: ich lebe auf einem boot (i live on a boat)
true_synonym: ich lebe auf einem schiff (i live on a ship)
false_friend: ich lebe auf einem schuh (i live on a shoe)

multilingual models mostly perform worse, bad ones even worse than chance, which indicates they are primarily good for english and prefer english representations.

May I add it to the german MTEB tasks and if so, what should I consider ? Is it a reranking task?

Thanks and all the best
Aaron

**Comment by KennethEnevoldsen** [comment]
Hi @achibb we very much welcome contributions.

There is a guide to adding a dataset [here](https://github.com/embeddings-benchmark/mteb/blob/main/docs/adding_a_dataset.md). Generally, we want a new dataset to cover something previously not covered by the benchmark. This indeed sound like such a case.

To me, it seems like a pair classification task where the pairs would be as follows:

(input sentence, true_synonym, label=positive)
(input sentence, false_friend, label=negative)

**Comment by KranthiGV** [comment]
I'm interested and would like to offer any help with this. Happy to collaborate or take on any tasks needed. Let me know!

**Comment by KennethEnevoldsen** [comment]
Wonderful @KranthiGV! I believe there are some potential good additions in the Aya and OpenAssistant datasets (large multilingual). If you start working on them please create an issue so that others can see that you are working on it. Alternatively, if you are less interested in dataset creation and more interested in improving the codebase then we also have a few tasks there.

@achibb do you have the time to add this dataset?

**Comment by achibb** [comment]
Yes it is on the plan for today or latest next week. Is this fine or does this intervene with some release ?

Gesendet von Outlook für iOS<https://aka.ms/o0ukef>
________________________________
Von: Kenneth Enevoldsen ***@***.***>
Gesendet: Friday, April 12, 2024 11:32:45 AM
An: embeddings-benchmark/mteb ***@***.***>
Cc: achibb ***@***.***>; Mention ***@***.***>
Betreff: Re: [embeddings-benchmark/mteb] Adding False Friends Dataset to German MTEB (Issue #331)


Wonderful @KranthiGV<https://github.com/KranthiGV>! I believe there are some potential good additions in the Aya and OpenAssistant datasets (large multilingual). If you start working on them please create an issue so that others can see that you are working on it. Alternatively, if you are less interested in dataset creation and more interested in improving the codebase then we also have a few tasks there.

@achibb<https://github.com/achibb> do you have the time to add this dataset?

**Comment by achibb** [comment]
The PR for false friends as pair classification model:

https://github.com/embeddings-benchmark/mteb/pull/349

tests so far look aligned with the research, even though the method is slightly edited. What do you think? Anything i need to do else? :) 
Thinking about doing such tests for other languages as well.

**Comment by KennethEnevoldsen** [comment]
Thanks @achibb - Will respond to the above in the PR

**Comment by achibb** [comment]
Nice thanks @KennethEnevoldsen I adjusted it except for two points where I have not found a use of the metadata in any file I looked.
So if given some feedback I would work it in but would also be fine for me to go with the current status. Also excited to see what @Muennighoff thinks in including it on run_mteb_german.py and benchmarks as I see this as a quite nice opportunity for adversarial testing :)
Enjoy the weekend guys!

**Comment by KennethEnevoldsen** [comment]
Yeah metadata in the old datasets is not great (we are working on it!), but the documentation for adding a new dataset have some examples.

Love the adversarial testing part! Would love to see more datasets like this one

**Comment by achibb** [comment]
Yeah no worries :). 

Yeah totally. I am also trying to contribute and am currently forming a group for multilingual stuff at hf.co/multilingual 

feel free to join anyone who wants and has time and would love to see some more languages creating false friends tests.

**Comment by KennethEnevoldsen** [comment]
Ahh that is great. Is it targeted at adversarial-style tests?

**Comment by KennethEnevoldsen** [comment]
The false friends dataset is available as `FalseFriendsGermanEnglish` and it is a part of `MTEB(deu, v1)`
